src/pytorch_lightning/loggers/wandb.py

```diff
@@ -339,6 +339,9 @@ def __init__(
         elif dir is not None:
             dir = os.fspath(dir)
```

Comment:
Can you add a CHANGELOG entry and a simple tests? You can update the file in https://github.com/Lightning-AI/lightning/blob/master/tests/tests_pytorch/loggers/test_wandb.py

Reply:
added tests